### PR TITLE
[Versioning] Update changelog entries for release 3.2.0.

### PR DESCRIPTION
**Pull request by gmarciani**
### Description of changes
Update changelog entries for release 3.2.0.

### Tests
Not required.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Signed-off-by: Giacomo Marciani <mgiacomo@amazon.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,11 +28,14 @@ x.x.x
 - Add `lambda:ListTags` and `lambda:UntagResource` to `ParallelClusterUserRole` used by ParallelCluster API stack for cluster update.
 - Add `parallelcluster:cluster-name` tag to all resources created by ParallelCluster.
 - Do not allow setting `PlacementGroup/Id` when `PlacementGroup/Enabled` is explicitly set to `false`.
-- Restrict IPv6 access to IMDS to root and cluster admin users only.
+- Restrict IPv6 access to IMDS to root and cluster admin users only, when configuration parameter `HeadNode/Imds/Secured` is enabled.
 - Change the default root volume size from 35 GiB to the size of AMIs. The default can be overwritten in cluster configuration file.
+- Disable the compute fleet when the configuration parameter `Scheduling/SlurmQueues/ComputeResources/SpotPrice`
```
Comment (enrico-usai) [inline CHANGELOG.md]: What about something like: "Automatic disabling of the compute fleet..."

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,11 +28,14 @@ x.x.x
...
+  is lower than the minimum required Spot request fulfillment price.
 
 **BUG FIXES**
 - Fix default for disable validate and test components when building custom AMI. The default was to disable those components, but it wasn't effective.
+- Handle corner case when instance is just launched and the describe doesn't report yet all the EC2 info.
```
Comment (enrico-usai) [inline CHANGELOG.md]: let's highlight it's a corner case in the scaling logic (and not for example a corner case in a CLI copmmand).

Comment (enrico-usai) [reply CHANGELOG.md]: describe --> describe instance API ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,9 +12,8 @@ x.x.x
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
 - Add support for FSx Lustre Persistent_2 deployment type.
 - Show `requested_value` and `current_value` values in the change set when adding or removing a section.
-- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
-  scheduling in Slurm.
-  - Add new configuration parameter to override default value of schedulable memory on compute nodes.
+- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
```
Comment (enrico-usai) [inline CHANGELOG.md:14]: These changes in the config parameters are more important than requested_value and current_value change.


Then the requested_value and current_value is more a change than an enhancement in my opinion. It has very low impact for the customer.

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,9 +12,8 @@ x.x.x
...
+- Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of schedulable memory on compute nodes.
```
Comment (enrico-usai) [inline CHANGELOG.md]: of schedulable memory --> of the compute nodes memory seen by the scheduler. 

What do you think?

Comment (gmarciani) [reply CHANGELOG.md]: Id' say `memory seen by the scheduler on compute nodes.`

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,10 +11,9 @@ x.x.x
 - Add support for multiple FSx File Systems.
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
 - Add support for FSx Lustre Persistent_2 deployment type.
-- Show `requested_value` and `current_value` values in the change set when adding or removing a section.
-- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
-  scheduling in Slurm.
-  - Add new configuration parameter to override default value of schedulable memory on compute nodes.
+- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
+- Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of the memory seen by the scheduler on compute nodes.
+
```
Comment (enrico-usai) [inline CHANGELOG.md]: extra newline

**Review by enrico-usai — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -11,10 +11,8 @@ x.x.x
 - Add support for multiple FSx File Systems.
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
```
Comment (mauri-melato) [inline CHANGELOG.md:12]: Let's use the proper name "FSx for NetApp ONTAP"
